.vscode/launch.json

```diff
@@ -10,7 +10,9 @@
       "type": "extensionHost",
       "request": "launch",
       "runtimeExecutable": "${execPath}",
-      "outFiles": ["${workspaceRoot}/**/*.bs.js"],
+      "outFiles": [
+        "${workspaceRoot}/_build/default/src/vscode_ocaml_platform.bc.js"
```

Comment:
I've tweaked the path to be `dist/` again in https://github.com/ocamllabs/vscode-ocaml-platform/pull/397